wandb/sdk/launch/docker.py

```diff
@@ -49,6 +50,7 @@ def generate_docker_image(project: _project_spec.Project, entry_cmd):
     cmd: Sequence[str] = [
         "jupyter-repo2docker",
         "--no-run",
+        "--user-id={}".format(project.uid),
```

Comment:
were you saying this still causes permissions issues? feel like it shouldn't with the r2d copy but we'd also have to sync with this uid on our copy until the double copy thing is fixed

Reply:
This line solves the permission issues created when a user sets the uid. Currently we chown with the same uid on the copy stuff, so they are synced.